src/service/resources-impl.js

```diff
@@ -341,6 +355,28 @@ export class ResourcesImpl {
     this.schedulePass();
   }
 
+  /**
+   * Select exp vs control for intersect-resources.
+   * @private
+   */
+  divertIntersectResources_() {
```

Comment:
Can we move this function into the same experiment file?

Reply:
done.